dissect/target/plugins/os/unix/config.py

```diff
@@ -81,54 +102,193 @@ def parse_file(self, fh: io.TextIO) -> None:
         self.parsed_data.read_file(open_file)
 
 
-class Txt(LinuxConfigurationParser):
+class Txt(ConfigurationParser):
     def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = {"content": fh.read(), "size": str(fh.tell())}
 
 
-class Default(LinuxConfigurationParser):
-    EMPTY_SPACE = re.compile(r"\s+")
+class Default(ConfigurationParser):
+    def __init__(
+        self,
+        collapse: Optional[Union[bool, set]] = False,
+        seperator: tuple[str] = (r"\s",),
+        comment_prefixes: tuple[str] = (";", "#"),
+    ) -> None:
+        super().__init__(collapse, seperator, comment_prefixes)
+        self.SEPERATOR = re.compile(rf"\s*?[{''.join(seperator)}]\s*?")
+        self.COMMENTS = re.compile(rf"\s*[{''.join(comment_prefixes)}]")
 
     def parse_file(self, fh: TextIO) -> None:
-        new_info = {}
+        information_dict = {}
+
+        skip_lines = self.comment_prefixes + ("\n",)
+        prev_key = None, None
         for line in fh.readlines():
-            if line.startswith(("#", "\n")):
+            if line.startswith(skip_lines):
```

Comment:
Why not:
```suggestion
            if line.strip().startswith(skip_lines):
```
and also be done with lines that just contain whitespace or whitespace preceiding a comment?